htlcswitch/hop/error_encryptor.go

```diff
@@ -80,9 +95,11 @@ type ErrorEncrypter interface {
 // result, all errors handled are themselves wrapped in layers of onion
 // encryption and must be treated as such accordingly.
 type SphinxErrorEncrypter struct {
-	*sphinx.OnionErrorEncrypter
+	encrypter interface{}
```

Comment:
Is it useful to define a marker interface in golang other than for documenting this specific field? I think it doesn't actually provide any compiler safety because every type will satisfy the interface?